examples/undocumented/python/graphical/lda.py

```diff
@@ -1,32 +1,31 @@
-from pylab import figure,pcolor,scatter,contour,colorbar,show,subplot,plot,connect
+import matplotlib.pyplot as plt
 from shogun import *
 import util
 
-util.set_title('LDA')
-util.DISTANCE=0.5
+plt.title('LDA')
+util.DISTANCE = 0.5
 
-gamma=0.1
+gamma = 0.1
 
 # positive examples
-pos=util.get_realdata(True)
-plot(pos[0,:], pos[1,:], "r.")
+pos = util.get_realdata(True)
+plt.plot(pos[0, :], pos[1, :], "r.")
 
 # negative examples
-neg=util.get_realdata(False)
-plot(neg[0,:], neg[1,:], "b.")
+neg = util.get_realdata(False)
+plt.plot(neg[0, :], neg[1, :], "b.")
 
 # train lda
-labels=util.get_labels()
-features=util.get_realfeatures(pos, neg)
-lda=LDA(gamma, features, labels)
+labels = util.get_labels()
+features = util.get_realfeatures(pos, neg)
+lda = LDA(gamma, features, labels)
```

Comment:
white we are at it. Could you port this to the new API usage, something in the lines of
```
lda = machine("LDA", gamma=gamma, labels=labels)
lda.train(features)
```